packages/blockchain-wallet-v4-frontend/src/components/Flyout/Brokerage/Trade.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import styled from 'styled-components'
+
+import { Button, Icon, Text } from 'blockchain-info-components'
+import { OptionRightActionRow } from 'components/Rows'
+
+const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+const IconWrapper = styled.div`
+  position: absolute;
+  top: 40px;
+  right: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
+  border: none;
+  background-color: ${({ theme }) => theme.grey100};
+  height: 32px;
+  width: 32px;
+`
+const ContentContainer = styled(Column)`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 128px;
+
+  & > div {
+    width: 100%;
+  }
+`
+const ButtonWrapper = styled.div`
+  display: flex;
+  align-items: flex-end;
+  margin-top: 40px;
+  padding: 0 40px 0 32px;
+
+  & > :first-child {
+    margin-right: 16px;
+  }
+`
+
+const rows = ({
+  handleDeposit,
+  handleReceive,
+  handleSend,
+  handleSwap,
+  handleWithdraw
+}): Array<RowType> => [
+  {
+    description: (
+      <FormattedMessage
+        id='modals.trade.swap.description'
+        defaultMessage='Exchange Crypto for Another'
+      />
+    ),
+    handleClick: handleSwap,
+    header: <FormattedMessage id='buttons.swap' defaultMessage='Swap' />,
+    iconColor: '#0C6CF2',
+    iconName: 'swap'
+  },
+  {
+    description: (
+      <FormattedMessage
+        id='modals.trade.send.description'
+        defaultMessage='Transfer to Another Wallet'
+      />
+    ),
+    handleClick: handleSend,
+    header: <FormattedMessage id='buttons.send' defaultMessage='Send' />,
+    iconColor: '#0C6CF2', // blue600
+    iconName: 'send'
+  },
+  {
+    description: (
+      <FormattedMessage
+        id='modals.trade.receive.description'
+        defaultMessage='Receive Crypto to Your Wallet'
+      />
+    ),
+    handleClick: handleReceive,
+    header: <FormattedMessage id='buttons.receive' defaultMessage='Receive' />,
+    iconColor: '#0C6CF2', // blue600
+    iconName: 'receive'
+  },
+  {
+    description: (
+      <FormattedMessage
+        id='modals.trade.receive.description'
+        defaultMessage='Add Cash to Your Cash Account'
+      />
+    ),
+    handleClick: handleDeposit,
+    header: <FormattedMessage id='buttons.deposit' defaultMessage='Deposit' />,
+    iconColor: '#0C6CF2', // blue600
+    iconName: 'deposit'
+  },
+  {
+    description: (
+      <FormattedMessage
+        id='modals.trade.receive.description'
+        defaultMessage='Cash Out to Your Bank Account'
+      />
+    ),
+    handleClick: handleWithdraw,
+    header: <FormattedMessage id='buttons.withdraw' defaultMessage='Withdraw' />,
+    iconColor: '#0C6CF2', // blue600
```

Comment:
there is a way to to use the color from theme. please use it in all spots above.  cc: @milan-bc